src/game/delete-round.ts

```diff
@@ -50,7 +51,8 @@ export class DeleteRound extends Round {
 
         const lines = new Array(game.state.lineLength).fill("");
 
-        lines[line] = this.getGameDeleteMeColumOffset(game) + "DELETE ME";
+        lines[line] =
+            (await this.getGameDeleteMeColumOffset(game)) + "DELETE ME";
```

Comment:
My only remaining comment is that you don't _really_ have to scope the method name as "game delete me" because the code is in the delete game's code already. I think `getColumnOffset` or even `columnOffset` would be fine.

No need to change it, just pointing it out.